docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -90,101 +114,91 @@ Consider the following HTML snippet:
-=== Sample Synthetic test
+=== Sample synthetic test
 
 A complete example of a basic synthetic test looks like this:
 
 [source,js]
 ----
 import { journey, step, expect } from '@elastic/synthetics';
 
-journey("Ensure placeholder is correct", ({page}) => {
-    step("go to todos demo page", async () => {
-      await page.goto('https://elastic.github.io/synthetics-demo/');
-    })
-    step("assert placeholder text", async () => {
-      const placeholderValue = await page.getAttribute(
-        'input.new-todo',
-        'placeholder'
-      );
-      expect(placeholderValue).toBe('What needs to be done?');
-    })
+journey('Ensure placeholder is correct', ({ page }) => {
+  step('Load the demo page', async () => {
+    await page.goto('https://elastic.github.io/synthetics-demo/');
+  });
+  step('Assert placeholder text', async () => {
+    const placeholderValue = await page.getAttribute(
+      'input.new-todo',
+      'placeholder'
+    );
+    expect(placeholderValue).toBe('What needs to be done?');
+  });
 });
 ----
 
 [discrete]
 [[synthetic-run-tests]]
-== Running synthetic tests
-
-As explained in the <<synthetics-quickstart,quickstart>>, there are two ways to run synthetic tests:
-
-* Run as an inline journey -- embed into `heartbeat.yml`
-* Run a test suite -- import your tests into Heartbeat
+== Run synthetic tests
 
-Which option is right for you? That depends.
-If you want to invoke a single journey with few steps, an inline journey might be the easiest workflow --
-you only need to write your tests into your `heartbeat.yml` file.
+There are two ways to run synthetic tests:
 
-Note that you can only paste the steps into an inline script, so these cannot be dependent on any packages that you may want to `import`. If you do have any dependencies, these would need to be defined with an `import` outside of the `journey` object and run via a test suite.
-If you have a complex test suite, or your tests need to live with your application code,
-you're likely better off setting up a synthetic test suite.
+* If you want to create a single journey and manage it in isolation from monitors containing other journeys,
+<<synthetics-inline-journey,use an inline journey>>.
+Inline journeys can be more efficient, but have some limitations like not supporting dependencies or multiple journeys.
+* If you have multiple journeys, rely on dependencies, or your tests need to live with your
+application code, <<synthetics-test-suite,use a test suite>>.
 
 [discrete]
 [[synthetics-inline-journey]]
-=== Run an inline journey
+=== Use an inline journey
 
 The easiest way to run a synthetic test is by creating an inline journey.
-The `journey` keyword isn't required, and access to variables like `page` and `params` is automatic. Remember you cannot `import` any dependencies using inline scripts.
+The `journey` keyword isn't required and access to variables like `page` and `params` is automatic.
+You cannot `import` any dependencies using inline scripts.
```

Comment:
Nit: Should this be in a separate warning callout rather than in the normal text flow?